openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -57,6 +57,37 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
     <div class="page-content">
         <div class="layout layout-1t2t">
             <main class="layout-col layout-col-b">
+
+                % if upgrade_url and upgrade_price:
+                <div class="section">
+                    <div class="vc-message tex2jax_ignore" role="group" aria-labelledby="vc-title" tabindex="-1" style="display: none;">
+
+                        <h3 id="vc-title" class="vc-fade vc-polite-only">${_("Pursue a verified certificate")}</h3>
+
+                        <button class="vc-toggle vc-fade vc-polite-only btn-link" type="button" aria-controls="moreinfo"
+                                aria-expanded="true" aria-label="${_("Show/Hide")}">
+                            ${_("Show less")}
+                        </button>
+
+                        <div class="vc-hero vc-fade">
+                            <img src="${static.url('course_experience/images/verified-cert.png')}"
+                                 alt="${_("Sample verified certificate with your name, the course title, the logo of the institution and the signatures of the instructors for this course.")}"/>
+                        </div>
+
+                        <ul class="vc-fade">
+                            <li>${_("Official proof of completion")}</li>
+                            <li>${_("Easily shareable certificate")}</li>
+                            <li>${_("Proven motivator to complete the course")}</li>
+                            <li>${_("Certificate purchases help edX continue to offer free courses")}</li>
```

Comment:
Don't include the string ``edX`` in any messages. Looking at this, I'm wondering whether this whole upsell should only be for edx.org, in which case it might better live in the edx.org theme directory. It probably is simpler to keep it but try to generalize it.

Reply:
Replaced "edX" with "us". If necessary, this can be modified in a future iteration.